src/meta/coinductive_predicates.lean

```diff
@@ -560,7 +574,7 @@ meta def coinductive_predicate (meta_info : decl_meta_info) (_ : parse $ tk "coi
 the quantifiers in the current goal.
 
 Current version: do not support mutual inductive rules (i.e. only a since C -/
```

Comment:
This comment seems to stop halfway a s

Reply:
Haha! Can you not imagine the end of the sentence? Sheesh!

Ok I'll fix that.